presto-orc/src/main/java/com/facebook/presto/orc/reader/SliceDictionarySelectiveReader.java

```diff
@@ -384,32 +388,41 @@ public Block getBlock(int[] positions, int positionCount)
             return new RunLengthEncodedBlock(outputType.createBlockBuilder(null, 1).appendNull().build(), positionCount);
         }
 
-        wrapDictionaryIfNecessary();
+        // compact values(ids) array, and calculate 1) the slice sizeInBytes if materialized, and 2) number of nulls
+        long blockSizeInBytes = 0;
+        int nullsCount = 0;  // the nulls count for selected positions
```

Comment:
naming consistency: nullsCount -> nullCount (to match positionCount and outputPositionCount)